tests/migrations/test_policies.py

```diff
@@ -103,3 +105,21 @@ def test_pending_migration_reverse(self, mock_get_status: Mock) -> None:
         # if forward migration was blocking, reverse not allowed even when pending
         migration_key = code_migration_key()
         assert NonBlockingMigrationsPolicy().can_reverse(migration_key) == False
+
+    @patch(
+        "snuba.migrations.runner.Runner.get_status",
+        return_value=(
+            Status.COMPLETED,
+            datetime.now() + timedelta(hours=-(MAX_REVERT_TIME_WINDOW_HRS - 5)),
+        ),
+    )
+    def test_completed_migration_reverse(self, mock_get_status: Mock) -> None:
```

Comment:
Would be a good idea to write some docstring to tell what the test is doing.

Reply:
added for both this test and the pending migration test